source/OptimaJet.DWKit.StarterApplication/Services/Auth0ManagementApiTokenService.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using static OptimaJet.DWKit.StarterApplication.Utility.EnvironmentHelpers;
+using RestSharp;
+using Newtonsoft.Json.Linq;
+using Serilog;
+
+namespace OptimaJet.DWKit.StarterApplication.Services
+{
+    public class Auth0ManagementApiTokenService
+    {
+        private string token;
+        public Auth0ManagementApiTokenService()
+        {
+        }
+
+        public string Token
+        {
+            get
+            {
+                if (token == null) {
+                    var domainUrl = GetVarOrThrow("AUTH0_DOMAIN");
+                    var clientId = GetVarOrDefault("AUTH0_TOKEN_ACCESS_CLIENT_ID", "");
+                    var clientSecret = GetVarOrDefault("AUTH0_TOKEN_ACCESS_CLIENT_SECRET", "");
+                    var client = new RestClient($"{domainUrl}/oauth/token");
+                    var request = new RestRequest(Method.POST);
+                    request.AddHeader("content-type", "application/json");
```

Comment:
the package you linked to is the AuthenticationAPI, rather than the ManagementAPI, not sure if that is intentional.

but anyway, this reasoning should be noted in the code, and an issue should be opened in Auth0's github and linked here. :)

Reply:
This code requests a token to use the Management API.  It looks like they are working on exposing the functionality in Auth0.net to do this request.  Their current documentation for C# to create this request is to use RestSharp (which we did).
https://auth0.com/docs/api/management/v2/tokens#automate-the-process
I can add some comments to the class to explain this.

So there is no need for an issue.  It is work in progress and not released code.